share/functions/fish_clipboard_copy.fish

```diff
@@ -13,4 +14,24 @@ function fish_clipboard_copy
     else if type -q clip.exe
         printf '%s' $cmdline | clip.exe
     end
+
+    # Copy with OSC 52. Primarily for SSH, but don't rely on SSH_TTY because
+    # that's dropped by su/sudo.
+    if not isatty stdout
+        echo "fish_clipboard_copy: stdout is not a terminal" >&2
+        return 1
+    end
+    set -l encoded (printf %s $cmdline | base64 | string join '')
```

Comment:
Technical not: `base64` is not in posix - it appears to be kind of common, tho.

I would not be opposed to a base64 `string escape` style.

Reply:
fwiw, `base64` isn't installed by default on FreeBSD.

For my own scripts, I have found `openssl enc base64` to be more universally available than `base64`. I can't remember if the `openssl` command has been pulled from the base installation in new macOS versions, but it was part of the base installation at least as of 10.10.